.github/workflows/validate-build.yml

```diff
@@ -0,0 +1,17 @@
+name: Validate Frontend Build
+run-name: Validate Frontend Build (kicked off by ${{ github.actor }})
+on:
+  pull_request:
+    branches:
+      - "main"
```

Comment:
Does this cause the build to be rerun for every repush to a PR, or only on its first creation?

Reply:
On create and on new commits. Found the [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request): 

> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened